src/test/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/resource/ReportResourceTest.kt

```diff
@@ -74,29 +80,20 @@ class ReportResourceTest : SqsIntegrationTestBase() {
 
       @Test
       fun `access forbidden with wrong role`() {
-        webTestClient.get().uri("/incident-reports/${existingReport.id}")
+        webTestClient.get().uri(url)
           .headers(setAuthorisation(roles = listOf("ROLE_BANANAS")))
           .header("Content-Type", "application/json")
           .exchange()
           .expectStatus().isForbidden
       }
-
-      @Test
```

Comment:
Is this test gone? I see there are new similar tests but not for the `GET /incident-reports/${existingReport.id}` endpoint? (I'm probably missing something)

Reply:
it's doing the wrong thing. `MAINTAIN_INCIDENT_REPORTS` is _not_ needed. and `read` is the _right_ scope (tho the endpoint does not require any particular scope)